entrypoint.sh

```
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
```

Shebang and `set -e` are correct and appropriate for an entrypoint. `set -e` ensures the container fails fast on errors which is what you want during startup.